pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -273,9 +273,38 @@ func ContainerFromNative(n *native.Container) (*Container, error) {
 		}
 		c.NetworkSettings = nSettings
 	}
+
+	c.Config = &Config{
+		Hostname: n.Labels[labels.Hostname],
+		Labels: filterLabels(n.Labels,
+			// filter out internal labels
+			hasPrefix(labels.Prefix),
+			hasPrefix("containerd.io"), // https://github.com/containerd/containerd/blob/release/1.6/runtime/restart/restart.go#L42-L50
+			hasPrefix(containerd.StopSignalLabel)),
+	}
```

Comment:
I don't think we should filter out internal labels.
Happy to discuss in a separate issue/PR though.